spec/controllers/questions/triage_lookback_controller_spec.rb

```diff
@@ -31,9 +31,9 @@
 
     context "when at least one situation is yes" do
```

Comment:
looks like this now tests basically the same thing as the next context block? unless i'm missing a meaningful difference you can probably delete this one